src/main/resources/lang/english.lang

```diff
@@ -1204,6 +1204,18 @@ entities:
 		name: strider¦s
 		pattern: <age> strider(|1¦s)|(4¦)strider (kid(|1¦s)|child(|1¦ren))
 
+# -- Heal Reasons --
+heal reasons:
+    custom: unknown, custom, plugin, a plugin
+    eating: eating, consuming, ingesting
+    ender_crystal: end crystal, ender crystal
```

Comment:
Inconsistency in the use of ``a``, if you want to use it throughout, you should probably have ``an end crystal`` and ``an ender crystal`` also, right?

Reply:
I'll add that